pkg/agent/openflow/client_test.go

```diff
@@ -90,7 +90,6 @@ func TestIdempotentFlowInstallation(t *testing.T) {
 		numFlows  int
 		installFn func(ofClient Client, cacheKey string) (int, error)
 	}{
-		{"NodeFlows", "host", 2, installNodeFlows},
```

Comment:
why it's removed?

Reply:
`TestIdempotentFlowInstallation` checks if `AddAll` is used only once when installing Pod/Node. Now, it is not true for Node: 1) it uses `BundleOps` instead of `AddAll`; 2) the method is applied for more than once.